etl/steps/data/garden/artificial_intelligence/2024-06-28/ai_adoption.meta.yml

```diff
@@ -0,0 +1,29 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Artificial Intelligence
+
+      grapher_config:
+              note: Companies represent a range of industries, sizes, functional specialties, and tenures. To adjust for differences in response rates, the data are weighted by the contribution of each respondent’s nation to global GDP.
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  ai_adoption:
+    variables:
+       pct_of_respondents:
+        title: Organizations reporting having adopted AI
+        description_short: Share of companies using AI technology (e.g., machine learning, computer vision, or natural language processing) in at least one business function, based on an average of 1,763 companies that responded to a global survey over those years.
```

Comment:
I don't fully understand where the 1763 comes from, but since the n for this survey seems to change each year (according to McK n=(1843, 1492, 1684, 1365) for 2021-24) I would argue giving the average here is error-prone and not super insightful - I would probably say something like (more than XXX companies or between XXX and YYY companies), but ultimately up to you.

Reply:
good point! I've removed it from the subtitle and added a note on this in description_key now! thanks!